projects/sbb-esta/angular-business/src/lib/header/header/header.component.scss

```diff
@@ -0,0 +1,84 @@
+@import '../styles/header';
+
+.sbb-header {
+  width: 100%;
+
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  z-index: 1000;
+  height: 54px;
+
+  background-color: white;
+  border-bottom: solid 1px #dcdcdc;
+
+  &-ribbon {
+    width: 80px;
+    position: absolute;
+    top: 12px;
+    left: -20px;
+    text-align: center;
+    line-height: 12px;
+    letter-spacing: 0px;
+    font-size: 10px;
+    color: #f0f0f0;
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
```

Comment:
Browser specific variants are not needed, since postcss will add them where necessary